### DISCUSSION TITLE
Возможность искать города по всем странам

### DISCUSSION THREAD

**Discussion by Asadul-Goba** [darkxanter/dadata_flutter#32] [General]
Здравствуйте, ваш проект dadata_flutter отлично работает. Единственное хотелось узнать можно ли сделать так что бы искать можно было не только города России но и всех остальных стран. На сайте dadata в документации написано что так можно сделать. Для этого нужно нужно добавить ограничение, тогда он начинает искать города по всему миру.

constraints: {
      locations: { country: "*" },
    }
    
Это код для джаваскрипта, можно ли это реализовать в вашем репозитории? Просто в дарте и флаттере я не очень силен, поэтому сам незнаю как это реализовать...

**Comment by darkxanter** [comment]
Вот так:
```dart
final resp = await widget.dadataClient.suggestAddress(
  AddressSuggestionRequest(
    text,
    constraints: [
      AddressSuggestionConstraint(country: '*'),
    ],
  ),
);
```
